packages/web3torrent/src/components/torrent-info/TorrentInfo.scss

```diff
@@ -50,6 +50,48 @@ section {
       min-width: 70px;
       width: 23%;
       text-align: right;
+      color: $c-orange;
+      font-weight: 500;
+      margin-top: 0px;
+      font-size: 18px;
+      position: relative;
+      outline:0;
+
+      .tooltiptext {
+        font-family: "Nunito Sans", sans-serif;
```

Comment:
Is this font being loaded somewhere?

Reply:
I thought that I did, but... now, seeing the code, I dont, I'll fix it

Reply:
umm... yeah, I did, I forgot, but I import it in the global App.scss